components/script/dom/document.rs

```diff
@@ -5603,6 +5608,49 @@ impl DocumentMethods<crate::DomTypeHolder> for Document {
     fn VisibilityState(&self) -> DocumentVisibilityState {
         self.visibility_state.get()
     }
+
+    fn CreateExpression(
```

Comment:
Though admittedly it is a bit silly in this case because there is no specification for https://dom.spec.whatwg.org/#dom-xpathevaluatorbase-createexpression as far as i can see.

Reply:
Will add these anyhow!